config/initializers/spotlight_initializer.rb

```diff
@@ -1,6 +1,6 @@
 Spotlight::Engine.config.upload_title_field = Spotlight::UploadFieldConfig.new(
-  field_name: %w(title title_full_display title_display title_245_search title_sort),
```

Comment:
Do we need to support the `title` field still?

Reply:
No, because its not a real solr field.